index.js

```diff
@@ -31,7 +31,10 @@ const activeFocusTraps = {
       trapStack.splice(trapIndex, 1);
     }
 
-    if (trapStack.length > 0) {
+    if (
+      trapStack.length > 0 &&
+      !trapStack[trapStack.length - 1]._isManuallyPaused()
+    ) {
       trapStack[trapStack.length - 1].unpause();
```

Comment:
Given we don't have symmetry here by calling `_setPausedState(false)` (opposite of line 14), I think we need a comment here to mention why we're calling `unpause()` instead of `_setPausedState(false)` -- that being to ensure the `manuallyPaused` state flag is reset on auto-unpause.

Then again, I think we could safely call `_setPausedState(false)` here to make it symmetric because we've already ensured that `state.manuallyPaused` is `false` with the `&& !trapStack[trapStack.length - 1]._isManuallyPaused()` check. That's probably much better than the comment.

Reply:
> Then again, I think we could safely call _setPausedState(false) here to make it symmetric because we've already ensured that state.manuallyPaused is false with the && !trapStack[trapStack.length - 1]._isManuallyPaused() check

Good point, so to confirm, we replace `trapStack[trapStack.length - 1].unpause();` with `_setPausedState(false)`, right?

Reply:
> > Then again, I think we could safely call _setPausedState(false) here to make it symmetric because we've already ensured that state.manuallyPaused is false with the && !trapStack[trapStack.length - 1]._isManuallyPaused() check
> 
> Good point, so to confirm, we replace `trapStack[trapStack.length - 1].unpause();` with `_setPausedState(false)`, right?

Yes! More specifically, with `trapStack[trapStack.length - 1]._setPausedState(false);`

Reply:
Updated